lib/rubocop/cop/lint/void.rb

```diff
@@ -106,6 +133,16 @@ def check_defined(node)
           add_offense(node, message: format(DEFINED_MSG, node.source))
         end
 
+        def check_nonmutating(node)
+          unless node.send_type? &&
+                 NONMUTATING_METHODS.include?(node.method_name)
+            return
+          end
+
+          add_offense(node, message: format(NONMUTATING_MSG, node.method_name,
```

Comment:
Please use a named fragment in the format string, e.g.:

```ruby
'Method `%<method_name>s` used in void context. Did you mean `%<method_name>s!`?'
```

and pass a hash to `format`, instead of repeating the argument. 🙂